maquetado/scss/styles.scss

```diff
@@ -1,4 +1,25 @@
-.title {
-  color: red;
+h1,
+p {
```

Comment:
giving style to elements adds to much specificity, try to use classes instead

Reply:
I thought we use `p` since it brings us a breakline. If I dont use the display: block I have to use <br>? Or how do I define each label is in a different line?